AccountData/src/main/java/skedgo/tripkit/account/data/UserKeyRepositoryImpl.kt

```diff
@@ -31,4 +34,7 @@ internal class UserKeyRepositoryImpl(
             .putString(KEY_USER_UUID, userKey)
             .apply()
       }
+          .andThen(Completable.fromCallable {
+            backupManager.dataChanged()
+          })
```

Comment:
> backupManager.dataChanged()

Should we call this when the userToken changes too?
If the userToken is already in SP, is there anyway that we can back it up?